src/child/enums/actions.js

```diff
@@ -0,0 +1,26 @@
+
+const ACTIONS = {
```

Comment:
Please correct me if I'm wrong, but from what I've read - an action is a plain JS object, with a `type` property - I think these are _action types_, not _actions_.